packages/lit-html/package.json

```diff
@@ -155,6 +152,127 @@
     "!/development/test/",
     "/directives/"
   ],
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts:types",
+        "build:version-stability-test"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo",
+        "tsconfig.polyfill-support.tsbuildinfo"
```

Comment:
Is this right? Doesn't it need to go with the command that uses `tsconfig.polyfill-support.json`? Which command is that, btw? This one because it's used as a project reference?

Reply:
Yes. Added a `"#comment"`.

I considered breaking this out into a separate build step, but they're sort of tangled up because they read and write to the same places.